src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -433,7 +433,7 @@ public static void ConfigureLogging()
 				builder.AddFilter("Uno.UI.RemoteControl", LogLevel.Information);
 
 				// Adjust logging when debugging the Given_HotReloadWorkspace tests
-				builder.AddFilter("Uno.UI.RuntimeTests.Tests.HotReload.Given_HotReloadWorkspace", LogLevel.Warning);
+				builder.AddFilter("Uno.UI.RuntimeTests.Tests.HotReload.Given_HotReloadWorkspace", LogLevel.Debug);
```

Comment:
Only in debug builds, otherwise there's too much logging in CI.

Reply:
Well that the purpose : we have no logs at all of what occurs on the CI for HR tests and it make that really difficult to diagnose.

(Note: Anyway this will be deprecated by https://github.com/unoplatform/uno/pull/15030)